android/src/main/java/com/swmansion/rnscreens/ScreenFragment.kt

```diff
@@ -247,19 +253,19 @@ open class ScreenFragment : Fragment {
         }
     }
 
-    fun registerChildScreenContainer(screenContainer: ScreenContainer<*>) {
-        mChildScreenContainers.add(screenContainer)
+    override fun addChildContainer(container: ScreenContainer) {
```

Comment:
Why do you change the method name without changing `mChildScreenContainers` name? What was wrong with the previous name?

Reply:
I just wanted to change prefixes: `register/unregister` -> `add/remove`, the `...Screen...` part should be there.

 https://github.com/software-mansion/react-native-screens/pull/1887/commits/e947d9ab9fde981ff955d632cee543ced8367e8c